extensions/common/context.cc

```diff
@@ -233,6 +278,11 @@ void populateHTTPRequestInfo(bool outbound, bool use_host_header_fallback,
   uint64_t response_flags = 0;
   getValue({"response", "flags"}, &response_flags);
   request_info->response_flag = parseResponseFlag(response_flags);
+
+  getValue({"request", "time"}, &request_info->start_time);
+  getValue({"request", "duration"}, &request_info->duration);
```

Comment:
I don't think this works with protobuf types. I believe these are bit-serialized as absl::Time and absl::Duration: https://github.com/envoyproxy/envoy-wasm/blob/master/source/extensions/common/wasm/context.cc#L302. Change the types of struct fields to match what the value should be.

Reply:
I see, it would be good to make the doc here more clear: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/rbac_filter.

Reply:
Yeah. We didn't want protobuf on the ABI, so the types changed to bit-representations whenever possible.

Reply:
To me it seems makes more sense to use primitive in abi (epoch, nanoseconds), but anyway.

Reply:
Good point. It might cause a bit of conversion overhead, but it's not set in stone.